cnv_release_notes/cnv_release_notes.adoc

```diff
@@ -123,6 +123,22 @@ $ virtctl image-upload ... image.raw.gz
 Restarting a virtual machine while it is in the *Importing* state results in an error.
 https://bugzilla.redhat.com/show_bug.cgi?id=1673921[(BZ#1673921)]
 
+* If the kubelet on a node crashes or restarts, this causes the kubelet to
+incorrectly report 0 KVM devices. Virtual machines are not properly scheduled
+on affected nodes.
++
+Verify the number of devices that the kubelet reports by running
+`$ oc get node $NODE | grep devices.kubevirt`. The output on an affected node
```

Comment:
by running:
    +
    ----
    $ oc get node $NODE | grep devices.kubevirt
    ----
    +
    The output . . . 

If you don't like that format, then you can keep the command as is, but drop the $

Reply:
Makes sense! Thanks for the suggestion. Fixed in https://github.com/openshift/openshift-docs/pull/13983/commits/368d71865fde6820944fb25ef98a83f6d9f4de9a